csvcubed-pmd/tests/behaviour/csvwcommandline.feature

```diff
@@ -57,6 +57,46 @@ Feature: Testing the csvw command group in the CLI
     And the file at "out/junior-roles.csv" should exist
     And the file at "out/gov.uk/schema/junior-roles.json" should exist
 
-    Scenario: The `pull` command should fail when the CSV-W file cannot be found.
+  Scenario: The `pull` command should fail when the CSV-W file cannot be found.
     When the pmdutils command CLI is run with "csvw pull https://example.com/non-existant-file.csv-metadata.json"
-    Then the CLI should fail with status code -1
\ No newline at end of file
+    Then the CLI should fail with status code 1
+
+
+  Scenario: "the 'uri replace' command should get and read in a ttl file but not be able to run without more arguments being entered in the command line"
```

Comment:
I'd move these tests into a separate `feature` file since they're not related to the `csvw` group of commands.